### PR TITLE
Add readthedocs config with docs requirements

### PR DESCRIPTION
**Pull request by RobbeSneyders**
Fixes #1390

https://docs.readthedocs.io/en/stable/faq.html#my-documentation-requires-additional-dependencies

### CODE REVIEW

File: .readthedocs.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 2
+
+sphinx:
+  configuration: docs/conf.py
+
+python:
+  version: 3.7
+  install:
+    - requirements: docs/requirements.text
```
Comment (hjacobs) [inline .readthedocs.yaml:9]:
```suggestion
    - requirements: docs/requirements.txt
```

**Comment by hjacobs** [comment]
I think you need to also install the `connexion` package as I ticked the "Install Project" box (which is then overwritten when having a config file):
Screenshot 2021-07-07 at 23-05-26 Edit Advanced Project Settings Read the Docs

**Comment by RobbeSneyders** [comment]
Good catch, thanks! Updated to install the package with `docs` extra. I also activated docs building on PRs to test the changes.

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**